packages/core/sandbox-layout/package.json

```diff
@@ -62,7 +62,6 @@
     "errorLimit": "200KB"
   },
   "peerDependencies": {
-    "@kong/kongponents": "^9.29.3",
```

Comment:
This is the only relevant change here right? If so I'm glad we are removing unnecessary peerDependencies 🎉